src/main/java/org/sagebionetworks/web/client/presenter/GridPagePresenter.java

```diff
@@ -0,0 +1,33 @@
+package org.sagebionetworks.web.client.presenter;
+
+import com.google.gwt.activity.shared.AbstractActivity;
+import com.google.gwt.event.shared.EventBus;
+import com.google.gwt.user.client.ui.AcceptsOneWidget;
+import com.google.inject.Inject;
+import org.sagebionetworks.web.client.DisplayConstants;
+import org.sagebionetworks.web.client.SynapseJSNIUtils;
+import org.sagebionetworks.web.client.place.GridPlace;
+import org.sagebionetworks.web.client.view.GridPageView;
+
+public class GridPagePresenter
+  extends AbstractActivity
+  implements Presenter<GridPlace> {
+
+  private final GridPageView view;
+
+  @Inject
+  public GridPagePresenter(GridPageView view, SynapseJSNIUtils jsniUtils) {
+    this.view = view;
+    jsniUtils.setPageTitle(DisplayConstants.WORKING_COPY);
```

Comment:
In the Synapse UI, we are currently planning to label grid sessions as 'working copies'

Reply:
Any way to know the source entity from the grid session Id?  Prefer the title to eventually be updated with something like "<entity-name> Working Copy"